tests/backends/test_mongo.py

```diff
@@ -0,0 +1,23 @@
+def test_schema_loader(context):
```

Comment:
It is more push/get test, not schema loader.

Reply:
Renamed.